demo/src/app/components/+typeahead/typeahead-section.list.ts

```diff
@@ -77,6 +78,17 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/dropup/dropup.html'),
         outlet: DemoTypeaheadDropupComponent
       },
+      {
+        title: 'On blur',
+        anchor: 'on-blur',
+        description: `
+         <p>Selects an option, on which user lost a focus. To reproduce start typing the name of the state, then focus
```

Comment:
Returns an option on which user lost a focus. To reproduce start typing the name of the state, then focus on one of the options with mouse or arrow keys and click outside of the typeahead.